diff -Naur p4c-df9e3ee28ae9b1d9d370c7b8543a953a1c490bc3.orig/backends/ebpf/runtime/kernel.mk p4c-df9e3ee28ae9b1d9d370c7b8543a953a1c490bc3/backends/ebpf/runtime/kernel.mk
--- p4c-df9e3ee28ae9b1d9d370c7b8543a953a1c490bc3.orig/backends/ebpf/runtime/kernel.mk	2024-11-27 22:26:12.000000000 +0100
+++ p4c-df9e3ee28ae9b1d9d370c7b8543a953a1c490bc3/backends/ebpf/runtime/kernel.mk	2024-11-28 13:25:17.438758022 +0100
@@ -11,7 +11,7 @@
 		-Wno-compare-distinct-pointer-types \
 		-Wno-gnu-variable-sized-type-not-at-end \
 		-Wno-address-of-packed-member -Wno-tautological-compare \
-		-Wno-unknown-warning-option -Wnoparentheses-equality
+		-Wno-unknown-warning-option -Wnoparentheses-equality -fno-stack-protector
 
 # Arguments for the P4 Compiler
 P4INCLUDE=-I$(ROOT_DIR)/p4include
