diff -Naur p4c-df9e3ee28ae9b1d9d370c7b8543a953a1c490bc3.orig/cmake/FindBMV2.cmake p4c-df9e3ee28ae9b1d9d370c7b8543a953a1c490bc3/cmake/FindBMV2.cmake
--- p4c-df9e3ee28ae9b1d9d370c7b8543a953a1c490bc3.orig/cmake/FindBMV2.cmake	2024-11-27 22:26:12.000000000 +0100
+++ p4c-df9e3ee28ae9b1d9d370c7b8543a953a1c490bc3/cmake/FindBMV2.cmake	2024-11-28 13:25:35.190829846 +0100
@@ -1,7 +1,8 @@
 set(BMV2_SIMPLE_SWITCH_SEARCH_PATHS
   ${CMAKE_INSTALL_PREFIX}/bin
   ${P4C_SOURCE_DIR}/../behavioral-model/build/targets/simple_switch
-  ${P4C_SOURCE_DIR}/../../behavioral-model/build/targets/simple_switch)
+  ${P4C_SOURCE_DIR}/../../behavioral-model/build/targets/simple_switch
+  CACHE STRING "")
 
 # check for simple_switch
 find_program (SIMPLE_SWITCH_CLI simple_switch_CLI
@@ -23,7 +24,8 @@
 set(BMV2_SIMPLE_SWITCH_GRPC_SEARCH_PATHS
   ${CMAKE_INSTALL_PREFIX}/bin
   ${P4C_SOURCE_DIR}/../behavioral-model/build/targets/simple_switch_grpc
-  ${P4C_SOURCE_DIR}/../../behavioral-model/build/targets/simple_switch_grpc)
+  ${P4C_SOURCE_DIR}/../../behavioral-model/build/targets/simple_switch_grpc
+  CACHE STRING "")
 
 # check for simple_switch_grpc
 find_program (SIMPLE_SWITCH_GRPC simple_switch_grpc PATHS ${BMV2_SIMPLE_SWITCH_GRPC_SEARCH_PATHS} )
@@ -40,7 +42,8 @@
 set(BMV2_PSA_SWITCH_SEARCH_PATHS
   ${CMAKE_INSTALL_PREFIX}/bin
   ${P4C_SOURCE_DIR}/../behavioral-model/build/targets/psa_switch
-  ${P4C_SOURCE_DIR}/../../behavioral-model/build/targets/psa_switch)
+  ${P4C_SOURCE_DIR}/../../behavioral-model/build/targets/psa_switch
+  CACHE STRING "")
 
 # check for psa_switch
 find_program (PSA_SWITCH_CLI psa_switch_CLI
@@ -62,7 +65,8 @@
 set(BMV2_PNA_NIC_SEARCH_PATHS
   ${CMAKE_INSTALL_PREFIX}/bin
   ${P4C_SOURCE_DIR}/../behavioral-model/build/targets/pna_nic
-  ${P4C_SOURCE_DIR}/../../behavioral-model/build/targets/pna_nic)
+  ${P4C_SOURCE_DIR}/../../behavioral-model/build/targets/pna_nic
+  CACHE STRING "")
   
 # check for pna_nic
 find_program (PNA_NIC_CLI pna_nic_CLI
 
